python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -183,6 +184,19 @@ cdef class RandomForest_impl():
         else:
             return CRITERION_END
 
+    def _get_max_feat_val(self):
+        if type(self.max_features) == int:
+            max_feature_val = 1/(self.max_features)
```

Comment:
When `max_features` is of `type int`, this number is the number of features to consider at each split, as per sklearn.
`max_feature_val` should be updated to `self.max_features/self.n_cols`